lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +43,21 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str) or len(text) == 0:
+            return None
+        tokens = []
+        alpha_cnt = 0
```

Comment:
done

Reply:
no, do not use additional variable `alpha_cnt`

Reply:
but the note says that this function should return None if no letters had been found. due to this, i thought checking just "if not tokens.." isnt going to be correct, because there might be just some punctuation signs and etc, but no letters...